eng/common/TestResources/deploy-test-resources.yml

```diff
@@ -1,76 +1,40 @@
-# Deploys resources to a cloud type specified by the variable (not parameter)
-# 'CloudType'. Use this as part of a matrix to deploy resources to a particular
-# cloud instance. Normally we would use template parameters instead of variables
-# but matrix variables are not available during template expansion so any
-# benefits of parameters are lost.
-
 parameters:
   ServiceDirectory: not-set
   ArmTemplateParameters: '@{}'
   DeleteAfterHours: 24
   Location: ''
+  SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
```

Comment:
Correct me if I'm wrong but isn't this change going to break any pipelines that set CloudType today? Should we add a step here that will set the SubscriptionConfiguration based on the CloudType parameter? At least temporarily until we can update them to use the the configuration?